c++/cpp2py/converters/map.hpp

```diff
@@ -5,13 +5,13 @@ namespace cpp2py {
 
   template <typename K, typename V> struct py_converter<std::map<K, V>> {
 
-    static PyObject *c2py(std::map<K, V> const &m) {
+    static PyObject *c2py(std::map<K, V> m) {
       PyObject *d = PyDict_New();
       for (auto &x : m) {
-        pyref k = py_converter<K>::c2py(x.first);
+        pyref k = py_converter<std::decay_t<K>>::c2py(std::move(x.first));
```

Comment:
why the decay ?
- py_converter should static_assert T is not a ref normally.

Reply:
I am not so much worrying about references, rather about `const`.
We specialize only `py_converter<double>` and not `py_converter<const double>` or example. People may have containers of `const value_type`.

Reply:
Ok. It does not hurt.